src/ixdat/readers/zilien.py

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Could you add a bit to the module docstring here about the quirks of Zilien files? 
For instance, can you define here the terminology `_series_headers` and `_column_headers` which is non-obvious, and what confused me last time. I think Zilien .tsv is kind of unique among data files I've seen in having both of these, aka a multi-line data column definition, so best to define this terminology for the reader of the code. 
Add (with changes as you see fit):
"""
Zilien tsv files are somewhat unique in having two lines to define each of the data columns. The first one is referred to as "**series header**" and explains what the data describes, and the second one is called "**column header**" and specifies the speficic column. Typically a series header will specify the measuring device (e.g. "iongauge value") or MS channel (e.g. "C0M2") and will apply for two or more column headers specify the quantity where the first is time ("Time [s]") and the subsequent are the corresponding value(s) ("Pressure [mbar]" or "M2-H2 [A]"). 
Newer zilien files can also include all the data from a coupled Biologic potentiostat. These are grouped under the **series header** "EC-lab".
"""